localization/pose_initializer/src/pose_initializer/pose_initializer_core.cpp

```diff
@@ -44,6 +45,8 @@ PoseInitializer::PoseInitializer() : Node("pose_initializer")
     stop_check_duration_ = declare_parameter<double>("stop_check_duration");
     stop_check_ = std::make_unique<StopCheckModule>(this, stop_check_duration_ + 1.0);
   }
+  localization_trigger_ = std::make_unique<LocalizationTriggerModule>(this);
```

Comment:
For planning simulation, this line should be enabled when `ndt_enabled = true`. And skip calling of activate/deactivate if this is nullptr.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1873/commits/093494b80c373f32596cbbbc5aa7a9400cb0baa2
resolved